cmd/lncli/commands.go

```diff
@@ -707,7 +701,41 @@ func closeChannel(ctx *cli.Context) error {
 		req.ChannelPoint.OutputIndex = 0
 	}
 
-	stream, err := client.CloseChannel(ctxb, req)
+	txidChan := make(chan string, 1)
+	wg := sync.WaitGroup{}
```

Comment:
nit: troughout the codebase, wait groups are usually declared `var wg sync.WaitGroup`